moped-editor/src/views/projects/projectView/ProjectTimeline.js

```diff
@@ -429,6 +431,23 @@ const ProjectTimeline = ({ refetch: refetchSummary }) => {
         <div style={{ width: "400px" }}>{milestone.milestone_description}</div>
       ),
     },
+    {
+      title: "Related phase",
+      field: "moped_milestone",
+      customSort: (a, b) => {
+        const aPhaseName = phaseNameLookup[a.moped_milestone.related_phase_id];
+        const bPhaseName = phaseNameLookup[b.moped_milestone.related_phase_id];
+        if (aPhaseName > bPhaseName) {
+          return 1;
+        }
+        if (bPhaseName < aPhaseName) {
```

Comment:
there's a lil bug here: L440 and L443 are the same test, so the sorting does not work when you try to reverse it. you just need to flip `<` to `>` in L443.

Reply:
< thank you >